+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase.'
+ message='Start executing testcase.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase.'
Thu Jun 29 15:35:18 2023 - INFO  - Start executing testcase.
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 25: sestatus: command not found
+ grep 'SELinux status'
+ grep enabled
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 26'
+ message='oe_test_selinux_sec_sestatus.sh line 26'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 26'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 26
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 27: sestatus: command not found
+ grep 'SELinuxfs mount'
+ grep /sys/fs/selinux
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 28'
+ message='oe_test_selinux_sec_sestatus.sh line 28'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 28'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 28
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 29: sestatus: command not found
+ grep 'SELinux root directory'
+ grep /etc/selinux
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 30'
+ message='oe_test_selinux_sec_sestatus.sh line 30'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 30'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 30
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
+ grep 'Loaded policy name'
oe_test_selinux_sec_sestatus.sh: line 31: sestatus: command not found
+ grep targeted
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 32'
+ message='oe_test_selinux_sec_sestatus.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 32'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 33: sestatus: command not found
+ grep 'Current mode'
+ grep enforcing
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 34'
+ message='oe_test_selinux_sec_sestatus.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 34'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 35: sestatus: command not found
+ grep 'Mode from config file'
+ grep enforcing
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 36'
+ message='oe_test_selinux_sec_sestatus.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 36'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 37: sestatus: command not found
+ grep 'Policy MLS status'
+ grep enabled
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 38'
+ message='oe_test_selinux_sec_sestatus.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 38'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 39: sestatus: command not found
+ grep 'Policy deny_unknown status'
+ grep allowed
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 40'
+ message='oe_test_selinux_sec_sestatus.sh line 40'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 40'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 40
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 41: sestatus: command not found
+ grep 'Memory protection checking'
+ grep actual
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 42'
+ message='oe_test_selinux_sec_sestatus.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 42'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ sestatus
oe_test_selinux_sec_sestatus.sh: line 43: sestatus: command not found
+ grep 'Max kernel policy version'
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_selinux_sec_sestatus.sh line 44'
+ message='oe_test_selinux_sec_sestatus.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_selinux_sec_sestatus.sh line 44'
Thu Jun 29 15:35:18 2023 - ERROR - oe_test_selinux_sec_sestatus.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'Finish testcase execution.'
+ message='Finish testcase execution.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish testcase execution.'
Thu Jun 29 15:35:18 2023 - INFO  - Finish testcase execution.
+ CASE_RESULT 0
+ case_re=0
+ test -z 10
+ test 10 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jun 29 15:35:18 2023 - ERROR - failed to execute the case.
+ exit 10
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_selinux_sec_sestatus.sh post_test'
+ message='oe_test_selinux_sec_sestatus.sh post_test'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'oe_test_selinux_sec_sestatus.sh post_test'
Thu Jun 29 15:35:18 2023 - INFO  - oe_test_selinux_sec_sestatus.sh post_test
